src/ape/contracts/base.py

```diff
@@ -18,11 +18,6 @@
     from ape.managers.networks import NetworkManager
 
 
-def _encode_address_args(*args):
-    # Convert higher level address types to str
-    return [arg.address if isinstance(arg, AddressAPI) else arg for arg in args]
```

Comment:
I am guessing you are going to add a converter that does this?

Reply:
we already have one!

Reply:
This was there as a stopgap because we didn't do the full conversion step inside of `ape/contracts` methods yet

Reply:
Next would come the elimination of `_encode_address_kwargs` as well

Reply:
`_encode_address_kwargs` removed in 875cd73